modules/rosa-deleting-cluster.adoc

```diff
@@ -66,11 +66,11 @@ State:                      ready
 Private:                    No
 Created:                    May 13 2022 11:26:15 UTC
 Details Page:               https://console.redhat.com/openshift/details/s/296kyEFwzoy1CREQicFRdZybrc0
-OIDC Endpoint URL:          https://rh-oidc.s3.us-east-1.amazonaws.com/1s5v4k39lhm8sm59m90mi0822o31844a <3>
+OIDC Endpoint URL:          https://d3gt1gce2zmg3d.cloudfront.net/20eh31t5vaasiidbumcjfbnknb6c6883 <3>
```

Comment:
It seems like the URL for staging.
The URL for production will be `dvbwgdztaeq9o.cloudfront.net`.